CliClient/locales/zh_CN.po

```diff
@@ -3288,14 +3289,14 @@ msgid ""
 "\n"
 "%s"
 msgstr ""
-"无法连接到Joplin Nextcloud应用。 请在“同步配置”中检查配置。 完整的错误信息"
+"无法连接到 Joplin NextCloud 应用。 请在“同步配置”中检查配置。 完整的错误信息"
 "是：\n"
 "\n"
 "%s"
 
 #: ReactNativeClient/lib/SyncTargetNextcloud.js:26
 msgid "Nextcloud"
-msgstr "Nextcloud"
+msgstr "NextCloud"
```

Comment:
The name is `Nextcloud`, not `NextCloud`. Please check with Nextcloud's use of their name. You might be thinking of ownCloud.

Reply:
I'm sorry for that stupid typo (QAQ) ...... I took it for granted, I will correct it soon.

Reply:
No worries. I wouldn't care, but Nextcloud doesn't like it when it's written incorrectly.